java/common/src/main/java/ConfigUtil.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
```

Comment:
This should be part of some package.

Reply:
Package created in common module. Will add subsequent packages shortly.